src/FondOfSpryker/Zed/CompanyUnitAddressApi/Business/Model/CompanyUnitAddressApi.php

```diff
@@ -121,7 +121,7 @@ public function get(int $idCompanyUnitAddress): ApiItemTransfer
             $companyUnitAddressTransfer
         );
 
-        if ($companyUnitAddressTransfer === null) {
+        if ($companyUnitAddressTransfer->getAddress1() === null) {
```

Comment:
Why?

Reply:
@daniel-rose The method getCompanyUnitAddressById cant return null. Therefore I check now the address1 field.